homeassistant/components/dialogflow/__init__.py

```diff
@@ -95,25 +96,56 @@ class DialogFlowError(HomeAssistantError):
 )
 
 
+def get_api_version(message):
+    """Help determine the version of Dialogflow request."""
+    if message.get('id') is not None:
+        return 'V1'
+
```

Comment:
Might be worth making V1 and V2 constants

Reply:
Done